rtb/verbali/interni/VI_2024-11-10_ver1.0.0.typ

```diff
@@ -0,0 +1,75 @@
+In seguito alla vittoria della candidatura per il capitolato C4 abbiamo provveduto a pianificare una e-mail per lunedì 11 novembre per concordare un incontro con l'azienda proponente. Abbiamo chiesto disponibilità dopo il "Diario di Bordo" e la lezione rovesciata in modo da avere qualche chiarimento anche dal professor Vardanega e avere più materiale sul quale discutere.
+
+== Sito per la documentazione
+Dopo alcune proposte stilistiche riguardo al sito dove pubblicare la documentazione abbiamo deciso di aggiornarci alla prossima riunione. Oltre a sistemare l'aspetto del sito bisogna aggiungere la logica che permette di collegare i link del sito ai documenti in pdf. Abbiamo quindi creato la repository *sweatunipd.github.io* che conterrà i file Python che aggiorneranno i link del sito in modo che aprano i documenti desiderati.
+
+== Backlog
+Abbiamo deciso di utilizzare il sistema di issue di GitHub come backlog. Poiché abbiamo pianificato di creare più repository a seconda dell'area di azione, ad esempio una per la documentazione e una per il sito che visualizzi i documenti, abbiamo convenuto a creare ua sola *board* che fungesse da backlog. In base alle attività vengono create le issue nelle rispettive repository, così da mantenere la separazione strutturale e poterle collegare alle "commit", ma vengono inserite nella stessa board così da essere tracciabili nello stesso spazio. Per mantenere questa metodologia abbiamo adottato le seguenti regole:
+- Dopo che si ha ragionato sulle azioni future, sempre durante la riunione si andranno a creare le issue in modo che tutti siano allineati sulle attività del backlog.
+- Le issue vanno create come già in accordo dal gruppo (descritto nel verbale interno del 04 novembre 2024 in mancanza al momento delle Norme di Progetto).
+- Il redattore del verbale dovrà popolare la tabella delle decisioni collegando i codici delle attività con le rispettive issue.
+- I nomi delle issue devono essere ben descrittivi perché saranno inseriti nella Tabella delle decisioni.
+- I codici delle attività sono strutturati nel seguente modo: *[X][\#ID issue]* dove X è una lettera che identifica univocamente la repository, seguita dall'ID della issue scritta con quattro cifre così da mantenere una lunghezza fissa fino alla issue numero 9999 per ciascuna repository. Le lettere che identificano le repository al momento sono *D* per la documentazione e *S* per il sito.
+
+== Aggiornamento GitHub Action
+Per correttezza nei confronti dell'azienda proponente che deve firmare i verbali esterni abbiamo deciso di non pubblicare questi ultimi fino a presa visione della controparte. Inoltre un passaggio della verifica dei documenti è la compilazione dei file Typst. Abbiamo pianificato quindi i seguenti cambiamenti per la GitHub Action:
+- Deve compilare tutti i file .typ di tutti i branch, non solo del main branch, che hanno subito modifiche. In questo modo evitiamo il sovraccarico con l'aumentare della documentazione.
```

Comment:
La action deve poter compilare anche il file Typst modificato in formato PDF per rendere il lavoro ai revisiori più semplice, non vengono compilati tutti i file